Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2538 -- Replace Search.gov results with public `content-search` results on Combined Search

Resolves [EREGCSC-2538](https://jiraent.cms.gov/browse/EREGCSC-2538)

**Description**

Removes Search.gov from front end of application.

As a result, the Resources column (right column) on the Search page will have identical results as the Subjects page.  Same number of results displayed the same way.

**This pull request changes:**

- No longer makes requests to `resources/search` or `resources/` on the Search page
- Instead, makes requests to `content-search`, just like the Subjects page
- Displays results on the Search page using same styles/components as they are displayed on the Subjects page using the `PolicyResults` component
- Removes almost all front end methods/tests/fixtures/artifacts related to Search.gov functionality
   - Notable exception: Schema Validation methods used with Cypress will remain.  These methods are used to validate the schema of responses from third party APIs, of which Search.gov results were the only thing being tested.  For now, we'll keep them in the codebase.
- Updates end to end tests to add coverage for new functionality

**Steps to manually verify this change:**

1. Visit [experimental deployment](https://5wumozc4z0.execute-api.us-east-1.amazonaws.com/dev1173)
2. Search for anything using the search box in the top right of the header (i.e. the normal way to get to the search page)
   - ex: [/search/q=MAT](https://5wumozc4z0.execute-api.us-east-1.amazonaws.com/dev1173/search/?q=MAT)
4. Ensure that the Resources column on the right works as expected
5. In another browser tab, search for the same query on the Subjects page and ensure that the results here look identical to the results on the Search page
   - ex: [/subjects/q=MAT](https://5wumozc4z0.execute-api.us-east-1.amazonaws.com/dev1173/subjects/?q=MAT)
6. Log in and ensure that both Public and Internal documents are being displayed on both the Search and Subjects page
